lambdas/data-migration1/package.json

```diff
@@ -37,6 +36,7 @@
     "pg": "^8.3.0"
   },
   "devDependencies": {
+    "@cumulus/api": "7.1.0",
```

Comment:
What is the main distinction between `dependencies` and `devDependencies`? Did it affect why you moved `@cumulus/api`?

Reply:
This sums it up better than I would: https://nodejs.dev/learn/npm-dependencies-and-devdependencies

> Development dependencies are intended as development-only packages, that are unneeded in production. For example testing packages, webpack or Babel.

With the changes that I made, `@cumulus/api` is no longer used directly by the lambda code, it's only used by unit test code, so it's a dev dependency for running tests but not for the code in production